docs/release-notes.rst

```diff
@@ -5,6 +5,205 @@ This document describes major installer-related changes in Fedora releases.
+version by default for all new devices and keep the LUKS version of existing ones. Use
+the kickstart support or Blivet GUI to select the LUKS version.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/pull/5395
+
+Remove libgnomekbd
+^^^^^^^^^^^^^^^^^^
+
+libgnomekbd is a C library used by Anaconda to display the keyboard
+preview widget. It is stuck in GTK 3 and X11 (libxklavier).
+
+Replace it with Tecla.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/pull/5417
+
+Remove screenshot support
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It was previously possible to take a screenshot of the
+Anaconda GUI by pressing a global hotkey. This was
+never widely advertised & rather hard to use for anything
+useful, as it was also necessary to manually extract the
+resulting screenshots from the installation environment.
+
+Furthermore, with many installations happening in VMs,
+it is usually more convenient to take a screenshot using
+the VM software anyway.
+
+By dropping screenshot support, we can remove dependency
+on the ``keybinder3`` library as well as the necessary
+screenshot code in the GUI.
+
+See also:
+    - https://github.com/rhinstaller/anaconda/pull/5255
+
+Changes in kickstart support
+----------------------------
+
+Remove support for NVDIMM namespaces
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All additional support for NVDIMM is being deprecated and removed, especially the support
+for the namespace reconfiguration. However, namespaces configured in the block/storage mode
+can be still used for the installation.
+
+The ``nvdimm`` kickstart command is deprecated and will be removed in future releases.
+
+See also:
+    - https://github.com/storaged-project/blivet/pull/1172
+    - https://github.com/pykickstart/pykickstart/pull/469
+    - https://github.com/rhinstaller/anaconda/pull/5353
+
+The installation program now correctly processes the proxy configuration (#2177219)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, the installation program did not correctly process the ``--proxy`` option of the
+``url`` Kickstart command or ``inst.proxy`` kernel boot parameter. As a consequence, you could
+not use the specified proxy to fetch the installation image. With this update, the issue
+is fixed and proxy works as expected.
+
+See also:
+    - https://bugzilla.redhat.com/show_bug.cgi?id=2177219
+    - https://github.com/rhinstaller/anaconda/pull/4828
+
+Remove and deprecate selected kickstart commands and options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following deprecated kickstart commands and options are removed:
+
+    - ``autostep``
+    - ``method``
+    - ``logging --level``
+    - ``repo --ignoregroups``
+
+The following kickstart options are deprecated:
+
+    - ``timezone --isUtc``
+    - ``timezone --ntpservers``
+    - ``timezone --nontp``
+    - ``%packages --instLangs``
+    - ``%packages --excludeWeakdeps``
```

Comment:
Formatting of the list looks inconsistent with F39 rel notes (on GH), I think there should be just
```
- item
```
(no itital padding)

Reply:
Fixed.